plugins/base/base.gradle.kts

```diff
@@ -0,0 +1,11 @@
+plugins {
+    `java-library`
+}
+
+description = """
+Base shuffleboard plugin that provides the default data types and widgets.
+""".trimMargin()
+
+dependencies {
+    foo
```

Comment:
Oops, was trying to see why it wasn't evaluating declarations like `compile` and `testRuntime`